tony-azkaban/src/test/java/com/linkedin/tony/azkaban/TestTonyJob.java

```diff
@@ -52,8 +52,15 @@ public void setup() {
 
   @Test
   public void testMainArguments() {
+    final String azkabanInputDataset = "AZKABAN_INPUT_DATASET";
+    final String azkabanOutputDataset = "AZKABAN_OUTPUT_DATASET";
+
     final Props jobProps = new Props();
     jobProps.put(TonyJobArg.HDFS_CLASSPATH.azPropName, "hdfs://nn:8020");
+    jobProps.put(TonyJob.WORKER_ENV_PREFIX + azkabanInputDataset,
```

Comment:
This doesn't seem to exercise the new code path, is that intentional? Should we add AZKABAN_INPUT_DATASET_JOB_PROP and AZKABAN_OUTPUT_DATASET_JOB_PROP to job props here?

Reply:
Thanks for catching it, fixed it.